### DISCUSSION TITLE
difference between 25vw and 25%

### DISCUSSION THREAD

**Discussion by SpeedFlying** [community/community#55638] [Programming Help] [answered]
I am new to HTML CSS and I found there are 25vw and 25% for styling elements and want to know the difference between 25vw and 25% in detail.

**Comment by toprater** [comment, marked as answer]
In CSS, both 25vw and 25% are units of measurement used to specify dimensions, exactly 25% of the width.
The key difference between 25vw and 25% is how they determine the reference for their calculations. 25vw is based on the viewport width, while 25% is based on the parent element's width.
